### DISCUSSION TITLE
Reverse search is adding numbers before command

### DISCUSSION THREAD

**Discussion by GabrielDeml** [marlonrichert/zsh-autocomplete#172] [Q&A] [answered]
When I do a reverse search using Ctrl-r there is a number added before the command. Is there a good way to fix this?

![image](https://user-images.githubusercontent.com/25163537/104823048-43cd0800-5815-11eb-888a-c31425861e73.png)

**Comment by marlonrichert** [comment, marked as answer]
That's a bug. I'll need to fix it.

**Reply by marlonrichert** [reply]
Fixed in db3370d4cf3223029a7a5e3a06ba6f529fe5fa77.

**Reply by GabrielDeml** [reply]
Thank you! That fixed it. I found this plugin today and I feel like I came out of the dark ages of auto compleate.
